subsys/mgmt/mcumgr/smp_udp.c

```diff
@@ -240,13 +242,19 @@ int smp_udp_open(void)
 int smp_udp_close(void)
 {
 #if CONFIG_MCUMGR_SMP_UDP_IPV4
-	k_thread_abort(&(configs.ipv4.thread));
-	close(configs.ipv4.sock);
+	if (configs.ipv4.sock >= 0) {
+		k_thread_abort(&(configs.ipv4.thread));
+		close(configs.ipv4.sock);
+		configs.ipv4.sock = -1;
+	}
 #endif
 
 #if CONFIG_MCUMGR_SMP_UDP_IPV6
-	k_thread_abort(&(configs.ipv6.thread));
-	close(configs.ipv6.sock);
+	if (configs.ipv4.sock >= 0) {
+		k_thread_abort(&(configs.ipv6.thread));
+		close(configs.ipv6.sock);
+		configs.ipv4.sock = -1;
```

Comment:
Shouldn't this be ipv6?

Reply:
Yes. Copy/paste 🤦‍♂️... Fixed and force-pushed.